Allow disuse of identifiers in a connection group (#723)
ietf-tapswg/api-drafts: draft-ietf-taps-interface.md

to the group of the Connection associated with this property. For a multi-streaming transport,
this limits the number of allowed streams.

> ### Private Session (#private-session}

Reviewer: ```suggestion
### Private Session (#private-session}

```

> ### Private Session {#private-session}

Reviewer: Interim discussion: rename to "isolate" instead of "private".

Name:
: privateSession

Type:
: Boolean

Default:
: false

> When set to true, this property will initiate new connections without using

Reviewer: I think "connections" should be capitalized throughout?

any cached information (such as session tickets or cookies) from previous

> connections that are not entangled with it. The residue of this connection will

Reviewer: ```suggestion
connections that are not entangled with it. Any state generated by this connection will
```

Connection will only be shared with entangled connections. Cloned Connections

> will use saved state from within the Connection Group.

Reviewer: Note that this not guarantee no leakage of information, since things like RTT estimates, etc, may be fully isolated.

> will use saved state from within the Connection Group.

Reviewer 2: For me, this sentence implies that the state generated by this new Connection will **not** be used by other Connections that are not in the same Connection Group. But maybe it's worth making this explicit.
